### DISCUSSION TITLE
Browser compatability list

### DISCUSSION THREAD

**Discussion by theChaosCoder** [livewire/livewire#1426] [4. General]
I could not find any info about browser compatability in the doc. 
I saw that ie11 is supported. But what are the minimum supported versions for Safari/Chrome/Firefox? 

Can I use babel to add support for IE9/IE10?

**Comment by donnyxray** [comment]
I'm interested in this too. Looking at my sentry logs there are quite some dated browsers which can't handle Livewire. Often people with cheap tablets - some which run Android versions as old as Android 4. 

While I have no intent to support such devices, I do want to detect them and redirect them to an informative page. (As well as get them out of my sentry logs.)

I'm not sure if parsing browser versions is the way to go though. Probably not. We should probably test for specific Javascript functions instead.

**Comment by elodieirdor** [comment]
Hey, I’m interested in this too. After migrating from v3 to v4, we’ve started seeing issues in Sentry on older browsers (mostly older iPhones).
It would be really helpful to let users know when their browser isn’t supported.

**Comment by dwightwatson** [comment]
Yeah, it looks like there a number of changes that affect older browsers in v4. 

* [Made one fix here with a string interpolation issue](https://github.com/livewire/livewire/pull/9742)
* [Identified another issue with negative lookbehind not working in Safari 15](https://github.com/livewire/livewire/blob/f1a47fd91d4bdd5b044b5410e26dfa0e0b4bb611/js/evaluator.js#L49)

Happy to keep making PRs to restore older browser support, but would be good to know if Livewire intends to support older browsers or is the intention that v4 is dropping this support?
